Rock/Web/UI/Controls/Grid/Grid.cs

```diff
@@ -1268,212 +1268,10 @@ protected void Actions_ExcelExportClick( object sender, EventArgs e )
             int rowCounter = 4;
             int columnCounter = 1;
 
-            if ( this.DataSourceAsDataTable != null )
-            {
-                DataTable data = this.DataSourceAsDataTable;
-
-                // print headings
-                foreach ( DataColumn column in data.Columns )
-                {
-                    worksheet.Cells[3, columnCounter].Value = column.ColumnName.SplitCase();
-                    columnCounter++;
-                }
-
-                // print data
-                foreach ( DataRow row in data.Rows )
-                {
-                    for ( int i = 0; i < data.Columns.Count; i++ )
-                    {
-                        worksheet.Cells[rowCounter, i + 1].Value = row[i].ToString().ConvertBrToCrLf();
-
-                        // format background color for alternating rows
-                        if ( rowCounter % 2 == 1 )
-                        {
-                            worksheet.Cells[rowCounter, columnCounter].Style.Fill.PatternType = OfficeOpenXml.Style.ExcelFillStyle.Solid;
-                            worksheet.Cells[rowCounter, columnCounter].Style.Fill.BackgroundColor.SetColor( Color.FromArgb( 240, 240, 240 ) );
-                        }
-                    }
-
-                    rowCounter++;
-                }
-            }
+            if ( DataSourceAsDataTable != null )
+                AddExportDataFromDataTable( worksheet, ref rowCounter, ref columnCounter );
```

Comment:
@MrUpsideDown If you plan on submitting pull requests regularly, I would recommend downloading CodeMaid (free, in the online VS store).  It will auto-align your code, add brackets to single-line if statements, remove unnecessary whitespace and more. 

![screenshot 2015-04-21 09 35 34](https://cloud.githubusercontent.com/assets/1210933/7253342/015cd3c8-e80a-11e4-8740-91a5f372877f.png)

You do need to change one thing in the default configuration to be consistent with Spark:  uncheck "Update #endregion tags with region name".
![screenshot 2015-04-21 09 38 45](https://cloud.githubusercontent.com/assets/1210933/7253375/45a4365c-e80a-11e4-888a-755ff3d99c32.png)